src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -31,11 +31,11 @@ protected function setUp()
     {
         parent::setUp();
 
-        if (!extension_loaded('mongo') && !extension_loaded('mongodb')) {
+        if (!extension_loaded('mongo') && (!extension_loaded('mongodb') || !class_exists('MongoDB\Client', false))) {
```

Comment:
For PHP7, I would check separately whether the PHP extension is installed or the userland library (mongodb/mongodb) in two different steps. Users which did not install the userland library may not understand that they also need to install it in addition to the PHP "mongodb" extension.